tailwind.config.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
       },
       colors: {
         sauced: {
-          orange: "hsla(19, 100%, 50%, 1)"
+          orange: "hsla(19, 100%, 50%, 1)",
+          light: "#FFEFE4"
```

Comment:
Suggest using tools like https://find-nearest-tailwind-colour.netlify.app to identify similar tailwind colors - for the above, the linked tool will fail linking `rose-100` as the color, however, looking at https://tailwindcss.com/docs/customizing-colors a similar enough color exists as `orange-100`

If the suggestion is not appropriate, it should be raised in the PR that new colors are being added to the design system, as the insights configuration is sometimes linked or copy pasted into other repos! 🍕

Reply:
It's a great suggestion, but several new color values are being introduced into the design system as we expand, and just to stay in uniform with what we already have in place, I can convert the color value into hsl using tools like https://htmlcolors.com/hex-to-hsl.